packages/sdk/wallets/wallet-walletconnect/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@celo/wallet-walletconnect",
+  "version": "0.0.1-dev",
+  "description": "Wallet Connect wallet implementation",
+  "author": "Celo",
+  "license": "Apache-2.0",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "keywords": [
+    "celo",
+    "blockchain",
+    "sdk"
+  ],
+  "scripts": {
+    "build": "tsc -b .",
+    "clean": "tsc -b . --clean",
+    "docs": "typedoc && ts-node ../../utils/scripts/linkdocs.ts wallet-walletconnect",
+    "test": "jest --runInBand",
+    "lint": "tslint -c tslint.json --project .",
+    "prepublishOnly": "yarn build"
+  },
+  "dependencies": {
+    "@celo/connect": "1.0.3-dev",
+    "@celo/utils": "1.0.3-dev",
+    "@celo/wallet-base": "1.0.3-dev",
+    "@celo/wallet-remote": "1.0.3-dev",
+    "@walletconnect/client": "^2",
```

Comment:
Any reason to not use a full version number here?

Reply:
Yeah WC v2 is still beta. Makes sense to to just leave it like this for now but will pin when we merge.